specification/resourceconnector/resource-manager/Microsoft.ResourceConnector/stable/2022-10-27/appliances.json

```diff
@@ -0,0 +1,1093 @@
+              "$ref": "#/definitions/applianceOperationsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List Appliances operations": {
+            "$ref": "./examples/AppliancesListOperations.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ResourceConnector/appliances": {
+      "get": {
+        "tags": [
+          "appliances"
+        ],
+        "operationId": "Appliances_ListBySubscription",
+        "summary": "Gets a list of Appliances in a subscription.",
+        "description": "Gets a list of Appliances in the specified subscription. The operation returns properties of each Appliance",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/applianceListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List Appliances by subscription": {
+            "$ref": "./examples/AppliancesListBySubscription.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ResourceConnector/telemetryconfig": {
+      "get": {
+        "tags": [
+          "appliances"
+        ],
+        "operationId": "Appliances_GetTelemetryConfig",
+        "summary": "Gets the telemetry config.",
+        "description": "Gets the telemetry config.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/applianceGetTelemetryConfigResult"
```

Comment:
>applianceGetTelemetryConfigResult [](http://example.com/codeflow?start=37&length=33)

Since the GET operation is on a resource collection, the response should be an array

Reply:
This is just a standard GET call, the response will always be a single key. Why would this need to be an array or a list? Not sure I understand where we refer to this as a resource collection. Could you elaborate a little more on why this should be an array?

Reply:
@rkmanda, let us know if you need any additional details to move forward. As @aditimalladi mentioned this API is not a resource collection and should not be an array.